src/IO/TinyPlyLoader/TinyPlyFileLoader.cpp

```diff
@@ -134,6 +135,7 @@ FileData* TinyPlyFileLoader::loadFile( const std::string& filename ) {
 
     file.read( ss );
 
+    if ( vertBuffer && vertBuffer->count != 0 )
```

Comment:
This test is always true if this line is reached as it is the negation of the test at line 93 that ends the function if true.
To remove the cppcheck warning, only the modification proposed at line 93 is needed.

Reply:
well right, I fist add this line to remove the warning, then modifiy the bogus test, we may add the following comment
```suggestion
    // here we know that  !( !vertBuffer || vertBuffer->count == 0 ), i.e. ( vertBuffer && vertBuffer->count != 0 )
```